python/sglang/srt/server_args.py

```diff
@@ -375,6 +375,7 @@ def add_cli_args(parser: argparse.ArgumentParser):
                 "marlin",
                 "gptq_marlin",
                 "awq_marlin",
+                "awq_turbomind",
```

Comment:
When the user is using an NVIDIA GPU and an AWQ checkpoint with a group size of 128, we should default to using TurboMind AWQ rather than Marlin AWQ.